js/dmp.playlist.js

```diff
@@ -96,7 +96,7 @@ dmp.playlist.loadPlaylist = function(file) {
  */
 dmp.playlist.handleErrors = function(e) {
   if(e.type == gapi.drive.realtime.ErrorType.TOKEN_REFRESH_REQUIRED) {
-    dmp.auth.autoRefreshAuth();
+    console.log("You have been signed out from your Google Account.");
```

Comment:
For some reasons I'm always getting this error when trying to create a playlist. To simulate it you can click the "Create Playlist" button on the right of the "Add audio files from Google Drive" or call `dmp.playlist.createNewPlaylist();` from the console.

Maybe the drive v2 client lib is not compatible with `auth2`? For me the following workaround works.
Line 63 before calling `gapi.client.drive.files.insert` do:

```js
var accessTokenObj = {};		
accessTokenObj.access_token = dmp.getAccessToken();		
accessTokenObj.token_type = "Bearer";		
accessTokenObj.expires_in = "3600";		
gapi.auth.setToken(accessTokenObj);
```

Then everything works... looks like drive v2 is looking for an auth token out of `auth` rather than the `auth2` lib. Any idea why?

Reply:
The Realtime API has fixed the issue.